<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>615</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QFrame" name="bar_frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>200</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideLeft</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Запись</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Сварщик</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Деталь</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Продолжительность</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignTop</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Время</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата</string>
         </property>
        </column>
       </widget>
       <widget class="QGraphicsView" name="graphicsView"/>
       <widget class="QTextEdit" name="textEdit"/>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="opaqueResize">
        <bool>false</bool>
       </property>
       <widget class="QFrame" name="bar_frame_right">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3"/>
         </item>
        </layout>
       </widget>
       <widget class="QGraphicsView" name="graphicsView_2">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionCreateFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionFileMerge"/>
    <addaction name="actionFileSave"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Команды</string>
    </property>
    <addaction name="actionReadDataBPI"/>
    <addaction name="actionClearBPI"/>
    <addaction name="action_TestEEPROM"/>
    <addaction name="actionResetPortBPI"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionDelete"/>
    <addaction name="actionEdit"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionOptionsShow"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAboutShow"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreateFile">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/1.png</normaloff>:/res/1.png</iconset>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/21.png</normaloff>:/res/21.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionFileMerge">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/26.png</normaloff>:/res/26.png</iconset>
   </property>
   <property name="text">
    <string>Объединить</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/24.png</normaloff>:/res/24.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="actionReadDataBPI">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/34.png</normaloff>:/res/34.png</iconset>
   </property>
   <property name="text">
    <string>Читать данные из БПИ</string>
   </property>
  </action>
  <action name="actionClearBPI">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/36.png</normaloff>:/res/36.png</iconset>
   </property>
   <property name="text">
    <string>Очистить память данных БПИ</string>
   </property>
  </action>
  <action name="action_TestEEPROM">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/10.png</normaloff>:/res/10.png</iconset>
   </property>
   <property name="text">
    <string>Тест микросхем EEPROM-памяти БПИ</string>
   </property>
  </action>
  <action name="actionResetPortBPI">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/20.png</normaloff>:/res/20.png</iconset>
   </property>
   <property name="text">
    <string>Сброс порта БПИ</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/19.png</normaloff>:/res/19.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/33.png</normaloff>:/res/33.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/2.png</normaloff>:/res/2.png</iconset>
   </property>
   <property name="text">
    <string>Выделить все...</string>
   </property>
  </action>
  <action name="actionOptionsShow">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/22.png</normaloff>:/res/22.png</iconset>
   </property>
   <property name="text">
    <string>Опции...</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/3.png</normaloff>:/res/3.png</iconset>
   </property>
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="actionAboutShow">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/11.png</normaloff>:/res/11.png</iconset>
   </property>
   <property name="text">
    <string>О Программе...</string>
   </property>
  </action>
  <action name="actionUserExit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/43.png</normaloff>:/res/43.png</iconset>
   </property>
   <property name="text">
    <string>Выход пользователя</string>
   </property>
  </action>
  <action name="actionUnitNumber">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/46.png</normaloff>:/res/46.png</iconset>
   </property>
   <property name="text">
    <string>№ Детали</string>
   </property>
  </action>
  <action name="actionAddGraph">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/29.png</normaloff>:/res/29.png</iconset>
   </property>
   <property name="text">
    <string>Добавить график</string>
   </property>
  </action>
  <action name="actionRemoveGraph">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/28.png</normaloff>:/res/28.png</iconset>
   </property>
   <property name="text">
    <string>Убрать график</string>
   </property>
  </action>
  <action name="actionClearCanvas">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/1.png</normaloff>:/res/1.png</iconset>
   </property>
   <property name="text">
    <string>Очистить холст</string>
   </property>
  </action>
  <action name="actionCopyGraph">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/18.png</normaloff>:/res/18.png</iconset>
   </property>
   <property name="text">
    <string>Копировать графики</string>
   </property>
  </action>
  <action name="actionRemoveSelGraph">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/19.png</normaloff>:/res/19.png</iconset>
   </property>
   <property name="text">
    <string>Удалить график</string>
   </property>
  </action>
  <action name="actionGraphArrange">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/5.png</normaloff>:/res/5.png</iconset>
   </property>
   <property name="text">
    <string>Расположение графиков</string>
   </property>
  </action>
  <action name="actionMaximizeToCanvas">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/30.png</normaloff>:/res/30.png</iconset>
   </property>
   <property name="text">
    <string>Во всю область </string>
   </property>
  </action>
  <action name="actionScaleHorizontal">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/31.png</normaloff>:/res/31.png</iconset>
   </property>
   <property name="text">
    <string>Растянуть по горизонтали</string>
   </property>
  </action>
  <action name="actionScaleVertical">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/32.png</normaloff>:/res/32.png</iconset>
   </property>
   <property name="text">
    <string>Растянуть по вертикали</string>
   </property>
  </action>
  <action name="actionStretchHorizontal">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/41.png</normaloff>:/res/41.png</iconset>
   </property>
   <property name="text">
    <string>Сжать по горизонтали</string>
   </property>
  </action>
  <action name="actionStretchVertical">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/42.png</normaloff>:/res/42.png</iconset>
   </property>
   <property name="text">
    <string>Сжать по вертикали</string>
   </property>
  </action>
  <action name="actionFitInView">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/40.png</normaloff>:/res/40.png</iconset>
   </property>
   <property name="text">
    <string>Установить актуальный размер</string>
   </property>
  </action>
  <action name="actionPreviewShow">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/27.png</normaloff>:/res/27.png</iconset>
   </property>
   <property name="text">
    <string>Предпросмотр</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/23.png</normaloff>:/res/23.png</iconset>
   </property>
   <property name="text">
    <string>Распечатать</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/25.png</normaloff>:/res/25.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
